example-app/SantokuApp/src/fixtures/msw/datas/eventData.ts

```diff
@@ -92,7 +96,7 @@ ReactとFirebaseを使用してWebアプリケーションを構築した開発
 ■参加資格
 XXRが全くわからない方
 `,
-    endDate: '2023-06-07T10:00:00Z',
+    endDate: '2023-01-01T10:00:00Z',
```

Comment:
固定値のうち、このイベントだけ1/1にしている理由はなんですか？
元の6/7で良さそうです。

```suggestion
    endDate: '2023-06-07T10:00:00Z',
```

Reply:
すみまさん。間違えました。
元に戻しました。

Reply:
修正commit: 873c39dca7fa91231eabc305f7f0f8e74f767d3f